docs/semantic-search.ipynb

```diff
@@ -51,337 +58,235 @@
+          "output_type": "execute_result"
         }
       ],
       "source": [
-        "row1 = dataset.documents.iloc[0:1].to_dict(orient=\"records\")[0]\n",
-        "dimension = len(row1['values'])\n",
-        "print(f\"These embeddings have dimension {dimension}\")"
+        "\n",
+        "index_name = \"semantic-search\"\n",
+        "\n",
+        "if not pc.has_index(index_name):\n",
+        "    pc.create_index_for_model(\n",
+        "        name=index_name,\n",
+        "        cloud=\"aws\",\n",
+        "        region=\"us-east-1\",\n",
+        "        embed={\n",
+        "            \"model\":\"multilingual-e5-large\",\n",
+        "            \"field_map\":{\"text\": \"chunk_text\"}\n",
+        "        }\n",
+        "    )\n",
+        "\n",
+        "# Initialize index client\n",
+        "index = pc.Index(name=index_name)\n",
+        "\n",
+        "# View index stats\n",
+        "index.describe_index_stats()"
       ]
     },
     {
-      "cell_type": "code",
-      "execution_count": 5,
+      "cell_type": "markdown",
       "metadata": {},
-      "outputs": [
-        {
-          "name": "stdout",
-          "output_type": "stream",
-          "text": [
-            "Here are some example questions in the data set:\n",
-            "\n",
-            "  - Why is a \"law of sciences\" important for our life?\n",
-            "  - Is it possible to format a BitLocker or FileVault protected drive?\n",
-            "  - Can formatting a hard drive stress it out?\n",
-            "  - Are the new Samsung Galaxy J7 and J5 worth their price?\n",
-            "  - I just watched an add for Indonesia 2026 World Cup bid in YouTube, is it viable?\n",
-            "  - I am an 18 year old college student. Is it a viable idea to play poker in order to pay for my college tuition?\n",
-            "  - If the French monarchy had never been abolished, who would be the current king/queen?\n",
-            "  - Who was the best French King?\n",
-            "  - How do I obtain a free United States phone number using the Internet?\n",
-            "  - What is the change in your opinion about PM Narendra Modi after demonetization of 1000 and 500 rupees currency notes?\n"
-          ]
-        }
-      ],
       "source": [
-        "print(\"Here are some example questions in the data set:\\n\")\n",
-        "for r in dataset.documents.iloc[0:10].to_dict(orient=\"records\"):\n",
-        "    print(\"  -\" + r['metadata']['text'])"
+        "## Creating our dataset"
       ]
     },
     {
       "cell_type": "markdown",
-      "metadata": {
-        "id": "ebd7XSamfMsC"
-      },
+      "metadata": {},
       "source": [
-        "## Creating an Index\n",
+        "We're working with a small subset of a large multilingual dataset called Tatoeba. Tatoeba consists of hundreds of thousands of sentence translation pairs, and sometimes serves as a benchmark for crosslingual semantic search capabilities.\n",
```

Comment:
I might share 1-2 pairs here so people can see what the data looks like.

Reply:
Added!